rhobot.spec

```diff
@@ -33,14 +33,20 @@ BuildRoot: %{_tmpdir}/%{name}-%{version}-%{release}
 # ln -s $(pwd)/ ./_build/src/github.com/cfpb/rhobot
 # export GOPATH=$(pwd)/_build:%{gopath}
 
-export GOPATH=$(pwd)/go
+export WORKDIR=$(pwd)
+export GOPATH=$(pwd)/gopath
 export GOROOT=$(pwd)/go
-export PATH=$(pwd)/go/bin:$PATH
-go build -o rhobot github.com/cfpb/rhobot
+export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
+
+go get github.com/tools/godep
+cd $GOPATH/src/github.com/cfpb/rhobot
+godep restore
+go build -o rhobot
+cp rhobot $WORKDIR
 
 %install
 install -d %{buildroot}%{_bindir}
-install -p -m 0755 ./rhobot %{buildroot}%{_bindir}/rhobot
+install -p -m 0755 rhobot %{buildroot}%{_bindir}/rhobot
```

Comment:
@frank0631 It might make sense to retain the `./rhobot` just to make sure that the right Rhobot executable is copied, correct?  Or does that not work?